templates/instantanswer/new_ia.tx

```diff
@@ -21,6 +21,52 @@
         <span class="clearfix"></span>
     </div>
     
+    <div id="signup-bg" class="modal-bg hide"></div>
+    <div id="signup-form" class="hide modal-wrapper">
+        <div>
+            <h3 class="label left"> Create an Account</h3>
+            <div class="right" id="signup-cancel">
+                 <i class="ddgsi ddgsi-close"></i>
+            </div>
+            <p class="clearfix first-p">Thanks - but you'll need to create an account (or <a class="login_newia" href="#">sign in</a>) before you can proceed. Don't worry, it will be quick!</p>
+            <p class="error-msg hide" id="signup-undef-error">An error occurred. Please try again</p>
```

Comment:
Can we get way more specific here? As far as I can tell I am doing everything right, but am getting this error.